dowhy/gcm/_noise.py

```diff
@@ -20,3 +40,108 @@ def compute_noise_from_data(causal_model: InvertibleStructuralCausalModel, obser
                     observed_data[get_ordered_predecessors(causal_model.graph, node)].to_numpy()))
 
     return result
+
```

Comment:
Looks like a similar issue to a previous PR. This part of _noise.py has already been reviewed and merged.

Reply:
Yes, I rebased it now.